src/frontend/Util_ROM.cpp

```diff
@@ -283,6 +287,85 @@ int LoadBIOS()
     return Load_OK;
 }
 
+int LoadROM(const QByteArray *romdata, QString archivefilename, QString romfilename, int slot)
```

Comment:
please keep the QT specific stuff outside of FrontendUtil because it's used by other frontends as well